lib/Test2/API.pm

```diff
@@ -110,7 +110,7 @@ use base 'Exporter';
 # This will make sure that Context.pm is loaded the first time this module is
 # imported, then the regular import method is swapped into place.
 sub import {
-    require Test2::API::Context unless $INC{'Test2/API/Context.pm'};
```

Comment:
I can reproduce that, although for this particular case it isn't going to matter since it's only run once.  Might as well drop this change then. I wonder if there are other places it would make sense to use this optimization.